src/main/java/htsjdk/samtools/SAMFileHeader.java

```diff
@@ -65,8 +67,8 @@
      * Ways in which a SAM or BAM may be sorted.
      */
     public enum SortOrder {
-
         unsorted(null),
+        unknown(null),
```

Comment:
While it's probably not important, it's good practice to add new enum values at the end, in case anyone relied on the previous ordering/ordinal values.